03-data_tour.Rmd

```diff
@@ -0,0 +1,23 @@
+# Data
+
+There are three major kinds of data available from BioDIGS: sample metadata, soil testing data, and genomics and metagenomics data. All of these are available for use in your classroom.
```

Comment:
```suggestion
There are currently three major kinds of data available from BioDIGS: sample metadata, soil testing data, and genomics and metagenomics data. All of these are available for use in your classroom.
```
in case we add climate data or other public data later!